core/src/modules/plugin.ts

```diff
@@ -350,12 +350,14 @@ export namespace plugin {
    */
   export async function replaceTemplateProfileVariables(
     string: string,
-    profile: Profile
+    profile: Profile,
+    strict = true
   ): Promise<string> {
     if (string.indexOf("{{") < 0) return string;
 
     const data = await getProfileData(profile);
-    return MustacheUtils.strictlyRender(string, data);
+    if (strict === true) return MustacheUtils.strictlyRender(string, data);
+    return MustacheUtils.render(string, data);
```

Comment:
This allows non-strict renders for things like returning mustached properties for calculated properties.  That means someone can now do things like calculate `if ("{{last_purchase_category}}" !== ""){...}`